automation/run-integration-tests.sh

```diff
@@ -66,11 +74,14 @@ function run_tests {
         --verbose --verbose \
         --log-level=DEBUG \
         --durations=5 \
-        --cov=$PYTHON27_PATH/libnmstate \
-        --cov=$PYTHON27_PATH/nmstatectl \
-        --cov-report=html:htmlcov-py27 \
+        --cov=\$($PYTHON_SITE_PATH_CMD)/libnmstate \
+        --cov=\$($PYTHON_SITE_PATH_CMD)/nmstatectl \
+        --cov-report=html:htmlcov-integ \
         tests/integration \
     ${nmstate_pytest_extra_args}"
+    if [ -n "$TRAVIS" ];then
+        docker_exec 'cd /workspace/nmstate && coveralls'
```

Comment:
Does coveralls really need to run in the container? It might also be possible to run it from outside the container since the coverage report should be available there as well since it is a shared volume.

Reply:
The reason to run it in docker is `coveralls` need access to the source of `libnmstate` where
outside docker does not. Sorry I forgot to mention that in commit message.